.github/workflows/open-datadog-agent-pr.yml

```
@@ -0,0 +1,75 @@
+name: Open datadog-agent PR
+run-name:
+on:
+  pull_request:
+    # types:
+      # - closed
+
+jobs:
+  open_bump_pr:
+    # if: github.event.pull_request.merged == true
```

Shouldn't these lines be uncommented?
```suggestion
    types:
      - closed

jobs:
  open_bump_pr:
    if: github.event.pull_request.merged == true
```

---

indeed, thanks for noticing! And fixed